litmus-portal/frontend/src/components/WelcomeModal/Stepper.tsx

```diff
@@ -0,0 +1,300 @@
+import Button from '@material-ui/core/Button';
+import Step from '@material-ui/core/Step';
+import { StepIconProps } from '@material-ui/core/StepIcon';
+import StepLabel from '@material-ui/core/StepLabel';
+import Stepper from '@material-ui/core/Stepper';
+import Typography from '@material-ui/core/Typography';
+import clsx from 'clsx';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import config from '../../config';
+import useActions from '../../redux/actions';
+import * as UserActions from '../../redux/actions/user';
+import { history } from '../../redux/configureStore';
+import { RootState } from '../../redux/reducers';
+import useStyles from './styles';
+import ModalPage from './Modalpage';
+import { Paper, TextField } from '@material-ui/core';
+import PasswordInput from '../PasswordInput/PasswordInput';
+import WelcomeModal from './index';
+function QontoStepIcon(props: StepIconProps) {
+  const classes = useStyles();
+  const { active, completed } = props;
+
+  return (
+    <div
+      className={clsx(classes.rootStepper, {
+        [classes.activeStepper]: active,
+      })}
+    >
+      {completed ? (
+        <div className={classes.circleStepper} />
+      ) : (
+        <div className={classes.circleStepper} />
+      )}
+    </div>
+  );
+}
+
+function getSteps() {
```

Comment:
You don't need an array of empty strings. You are already passing activeStep from the main return block below